doc/server-secrets.md

```diff
@@ -40,25 +62,25 @@ rotation logic.
 [github rate limit]: https://developer.github.com/v3/#rate-limiting
 [personal access tokens]: https://github.com/settings/tokens
 
-* `gh_client_id`
-* `gh_client_secret`
+* `GH_CLIENT_ID` (yml: `gh_client_id`)
+* `GH_CLIENT_SECRET` (yml: `gh_client_secret`)
 
 These settings are used by shields.io for GitHub OAuth app authorization
 but will not be necessary for most self-hosted installations. See
 [production-hosting.md](./production-hosting.md).
 
 ## Jenkins CI
 
-* `jenkins_user`
-* `jenkins_pass`
+* `JENKINS_USER` (yml: `jenkins_user`)
+* `JENKINS_PASS` (yml: `jenkins_pass`)
 
 Provide a username and password to give your self-hosted Shields installation
 access to a private Jenkins CI instance.
 
 ## JIRA
 
-* `jira_user`
-* `jira_pass`
+* `JIRA_USER` (yml: `jira_user`)
+* `JIRA_PASS` (yml: `jira_pass`)
```

Comment:
I think I've confused myself as to whether the legacy format `jira_username` is still supported/if we're planning to support it 🤔

Reply:
If we're going to drop it, now is a good time since we're breaking BC anyway